src/commands/ExecCommand.js

```diff
@@ -38,6 +38,7 @@ export default class ExecCommand extends Command {
   getOpts(pkg) {
     return {
       cwd: pkg.location,
+      shell: true,
```

Comment:
`lerna exec -- npm view \$LERNA_PACKAGE_NAME` works perfectly fine as-is on `master`, for example.